### PR TITLE
oob ud: fixes and parameter adjustment

### PR DESCRIPTION
**Pull request by nkogteva**

### CODE REVIEW

File: orte/mca/oob/ud/oob_ud_component.c [hunk 1]
```diff
@@ -223,45 +222,40 @@ static inline int mca_oob_ud_device_setup (mca_oob_ud_device_t *device,
     /* If fork support is requested, try to enable it */
     rc = opal_common_verbs_fork_test();
     if (OPAL_SUCCESS != rc) {
-        opal_output_verbose(5, orte_oob_base_framework.framework_output,
-                            "%s oob:ud:device_setup failed in ibv_fork_init. errno = %d",
-                            ORTE_NAME_PRINT(ORTE_PROC_MY_NAME), errno);
+        opal_output(0, "%s oob:ud:device_setup failed in ibv_fork_init. errno = %d",
```
Comment (jsquyres) [inline orte/mca/oob/ud/oob_ud_component.c]: If this is intended to convey an error message to the end user (vs. an OMPI developer), opal_show_help() should be used instead.

Ditto for all the opal_output()'s in this PR.

**Comment by mellanox-github** [comment]
* Coverity found 911 errors for all: http://bgate.mellanox.com:8888/jenkins/job/gh-ompi-master-pr//ws/cov_build/all_449/output/errors/index.html

* Coverity found 5 errors for oshmem: http://bgate.mellanox.com:8888/jenkins/job/gh-ompi-master-pr//ws/cov_build/oshmem_449/output/errors/index.html

* Coverity found 2 errors for fca: http://bgate.mellanox.com:8888/jenkins/job/gh-ompi-master-pr//ws/cov_build/fca_449/output/errors/index.html <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by mellanox-github** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://bgate.mellanox.com/jenkins/job/gh-ompi-master-pr/449/
Test PASSed.

**Comment by nkogteva** [comment]
@jsquyres Since review of whole component is needed in order to replace all similar outputs I propose to use another one PR. Otherwise it would be difficult to see the difference between formal changes and real fixes in the same PR.

**Comment by jsquyres** [comment]
@nkogteva That would be fine, but then you should strip all the opal_output_verbose -> opal_output conversions from this PR.  Specifically: it does not make sense to me to commit something that makes a situation worse with the rationale that it will be fixed in the future (because that inevitably does not happen).  Even more specifically: it seems like this PR is two different things, anyway: some fixes and some opal_output_verbose->opal_output conversions.  So strip those out and make a 2nd PR to comprehensively convert all opal_output*'s of errors into opal_show_helps.

**Comment by mellanox-github** [comment]
* Coverity found 911 errors for all: http://bgate.mellanox.com:8888/jenkins/job/gh-ompi-master-pr//ws/cov_build/all_453/output/errors/index.html

* Coverity found 5 errors for oshmem: http://bgate.mellanox.com:8888/jenkins/job/gh-ompi-master-pr//ws/cov_build/oshmem_453/output/errors/index.html

* Coverity found 2 errors for fca: http://bgate.mellanox.com:8888/jenkins/job/gh-ompi-master-pr//ws/cov_build/fca_453/output/errors/index.html <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by jsquyres** [comment]
@nkogteva Thank you!

**Comment by mellanox-github** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://bgate.mellanox.com/jenkins/job/gh-ompi-master-pr/453/
Test PASSed.

**Comment by miked-mellanox** [comment]
:+1:

**Pull request merged**